bindings/java/src/org/sleuthkit/datamodel/blackboardutils/attributes/GeoTrackPointList.java

```diff
@@ -0,0 +1,293 @@
+/*
+ * Sleuth Kit Data Model
+ *
+ * Copyright 2020 Basis Technology Corp.
+ * Contact: carrier <at> sleuthkit <dot> org
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *	 http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.sleuthkit.datamodel.blackboardutils.attributes;
+
+import com.google.gson.Gson;
+import com.google.gson.annotations.SerializedName;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.sleuthkit.datamodel.blackboardutils.attributes.GeoWaypointList.GeoWaypoint;
+
+/**
+ * Helper class to make it easier to serialize and deserialize the
+ * TSK_GEO_TRACKPOINTS attribute.
+ *
+ */
+public final class GeoTrackPointList {
+
+	private final List<GeoTrackPoint> points;
+
+	/**
+	 * Deserialize the given list of GeoTrackPoints.
+	 *
+	 * @param jsonString JSon string of track points.
+	 *
+	 * @return	Timestamp ordered list of GeoTrackPoints, empty list will be
+	 *         returned if jsonString is null or empty.
+	 */
+	public static GeoTrackPointList deserialize(String jsonString) {
```

Comment:
The abstraction is actually to/from a BlackboardAttribute with type TSK_GEO_TRACKPOINTS. Could this method be changed to 'from(BlackboardAttribute attribute)', and throw an illegal arg if its not type TSK_GEO_TRACKPOINTS?